package.json

```diff
@@ -3,10 +3,7 @@
   "description": "A Secure NavCoin Wallet",
   "author": "NavPay",
   "version": "4.1.0",
-  "keywords": [
-    "navocin",
-    "wallet"
-  ],
+  "keywords": ["navcoin", "wallet", "nav", "NavPay"],
```

Comment:
why do we do a lowercase navcoin but mixed case NavPay ?

Reply:
No reason :)

Reply:
all the keywords are belong to us